tests/sentry/release_health/test_tasks.py

```diff
@@ -88,6 +96,9 @@ def setUp(self):
             group_id=self.event.group.id, project_id=self.project.id, release_id=self.release.id
         )
 
+    def tearDown(self):
+        self.backend.__exit__()
```

Comment:
Just curious, how did you know the `__exit__` takes three params?

Reply:
It's a standard thing for context managers https://docs.python.org/3/reference/datamodel.html#object.__exit__. tbh I don't know why I didn't add it in initially, I think I was just testing whether it would work without and for some reason the test passed individually, I guess because the teardown didn't matter.